src/pivotal-ui-react/sortable-table/sortable-table.js

```diff
@@ -44,100 +60,91 @@ var TableRow = React.createClass({
  * @component SortableTable
  * @description A table that can be sorted by column
  *
- * @property columns {Array<Object>} A list of column configuration parameters:
- *   `name` is the `data` key associated with the column;
- *   `title` is the header text for the column;
- *   `sortable` (defaults to false) indicates whether the table can be sorted by this column; and
- *   `align` (defaults to left) sets the text alignment for cells in this column
- * @property data {Array<Object>} The data to be displayed in the table
- * @property classes {Array<String>} Class names to add to the table
+ * @property `headers` {Array<Object>} A list of `TableHeader` components
  *
  * @example ```js
- * var SortableTable = require('pui-react-sortable-table').SortableTable;
+ * var {SortableTable, TableHeader, TableRow, TableCell} = require('pui-react-sortable-table');
  * var MyComponent = React.createClass({
  *   render() {
- *     var columns = [
- *       {name: 'c1', title: 'Column 1', sortable: true},
- *       {name: 'c2', title: 'Column 2'}
- *     ];
+ *     var headers = [
+ *       <TableHeader sortable={true}>c1</TableHeader>,
+ *       <TableHeader sortable={true}>c2</TableHeader>,
+ *      ];
  *     var data = [
  *       {c1: 'yes', c2: 'foo'},
  *       {c1: 'no', c2: 'bar'}
  *     ];
- *     return <SortableTable data={data} columns={columns} classes={['my-table']}/>;
+ *     return <SortableTable headers={headers}>
+ *       {sortTableData.map(function(datum, key) {
+ *         return (
+ *           <TableRow key={key}>
+ *             <TableCell>{datum.c1}</TableCell>
+ *             <TableCell>{datum.c2}</TableCell>
+ *           </TableRow>
+ *         );
+ *       })}
+ *     </SortableTable>;
  *   }
  * });
  * ```
  *
  * @see [Pivotal UI React](http://styleguide.pivotal.io/react.html#table_sortable_react)
  */
-var SortableTable = React.createClass({
+export const SortableTable = React.createClass({
   propTypes: {
-    classes: types.arrayOf(types.string),
-    columns: types.arrayOf(types.shape({
-      name: types.string.isRequired,
-      title: types.string.isRequired,
-      align: types.oneOf(['left', 'center', 'right']),
-      sortable: types.bool
-    })),
-    data: types.arrayOf(types.object)
+    headers: types.arrayOf(types.element)
   },
 
   getInitialState() {
-    var {columns, data} = this.props;
-    var column = columns[0].name;
-    return {
-      sort: {
-        column,
-        order: 'asc'
-      },
-      data: sortBy(data, column)
-    };
+    return {sortColumn: 0, sortAscending: true};
```

Comment:
Also, a user should be able to choose the default sort column/order. Not sure exactly how this should look.